add conda-forge recipe

@nehaljwani Even with those changes, I run into the error:
```
    C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -I\Users\User\usr\include\eigen-3.3.7\ -IC:\Users\User\AppData\Local\Temp\pip-build-env-835_y16g\normal\Lib\site-packages\pybind11\include -IC:\Users\User\AppData\Local\Temp\pip-build-env-835_y16g\normal\Lib\site-packages\pybind11\include -IC:\Users\User\Anaconda3\include -IC:\Users\User\Anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.22.27905\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.7.2\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /EHsc /Tpsrc/generate.cpp /Fobuild\temp.win-amd64-3.7\Release\src/generate.obj -std=c++14
    cl : Command line warning D9002 : ignoring unknown option '-std=c++14'
    generate.cpp
    src/generate.cpp(5): fatal error C1083: Cannot open include file: 'CGAL/Exact_predicates_inexact_constructions_kernel.h': No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.22.27905\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\User\Anaconda3\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\User\\usr\\include\\pygalmesh\\setup.py'"'"'; __file__='"'"'C:\\Users\\User\\usr\\include\\pygalmesh\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' develop --no-deps Check the logs for full command output.
```

I think we're almost there: the build runs as expected for linux and osx and the only thing preventing it to run on windows is the test tolerance in `test_volume_from_surface`:

```
        tol = 2.0e-3
        assert abs(max(mesh.points[:, 0]) - 0.357612477657) < tol
        assert abs(min(mesh.points[:, 0]) + 0.358747130015) < tol
>       assert abs(max(mesh.points[:, 1]) - 0.496137874959) < tol
E       assert 0.014181661246307764 < 0.002
E        +  where 0.014181661246307764 = abs((0.4819562 - 0.496137874959))
E        +    where 0.4819562 = max(array([-0.19258097, -0.33402   , -0.3258206 ,  0.06994132, -0.1434645 ,\n       -0.42768064,  0.05553488,  0.0659714 , ...5258,  0.37050304, -0.1880949 ,  0.39482927,\n       -0.10133088,  0.07262085, -0.20886302, -0.19134675], dtype=float32))

```
https://dev.azure.com/conda-forge/84710dde-1620-425b-80d0-4cf5baca359d/_apis/build/builds/153741/logs/30

@nschloe if you up the tolerance to 2.0e-2 it seems that we should be able to have builds for the three platforms possible!

--- a/README.md
+++ b/README.md
@@ -7,6 +7,7 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat-square)](https://github.com/psf/black)
 [![PyPI pyversions](https://img.shields.io/pypi/pyversions/pygalmesh.svg?style=flat-square)](https://pypi.org/pypi/pygalmesh/)
 [![PyPi Version](https://img.shields.io/pypi/v/pygalmesh.svg?style=flat-square)](https://pypi.org/project/pygalmesh)
+[![Anaconda Cloud](https://anaconda.org/conda-forge/pygalmesh/badges/version.svg?=style=flat-square)](https://anaconda.org/conda-forge/pygalmesh/)
 [![Debian CI](https://badges.debian.net/badges/debian/testing/python3-pygalmesh/version.svg?style=flat-square)](https://tracker.debian.org/pkg/python3-pygalmesh)
 [![GitHub stars](https://img.shields.io/github/stars/nschloe/pygalmesh.svg?style=flat-square&label=Stars&logo=github)](https://github.com/nschloe/pygalmesh)
 [![PyPi downloads](https://img.shields.io/pypi/dm/pygalmesh.svg?style=flat-square)](https://pypistats.org/packages/pygalmesh)

